docs/Getting_Started.md

```diff
@@ -0,0 +1,421 @@
+# Getting started with Discord app development
+
+Discord apps let you customize your servers with interactions and automation. This guide is meant to walk through building and running your first Discord app using JavaScript. At the end of this guide, you’ll have an app with a bot user that uses slash commands, sends messages, and interacts with message components.
+
+While this guide is beginner-focused, it assumes a basic understanding of [JavaScript](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics).
+
+> info
+> When developing apps, you should build and test in a server that isn’t actively used by others. If you don’t have your own server already, you can [create one for free](https://support.discord.com/hc/en-us/articles/204849977-How-do-I-create-a-server-).
+
+### Guide contents
+- [Overview](#DOCS_GETTING_STARTED/overview)
+- [Creating an app](#DOCS_GETTING_STARTED/creating-an-app)
+- [Running your app](#DOCS_GETTING_STARTED/running-your-app)
+- [Handling interactivity](#DOCS_GETTING_STARTED/handling-interactivity)
+- [Adding message components](#DOCS_GETTING_STARTED/adding-message-components)
+- [Next steps](#DOCS_GETTING_STARTED/next-steps)
+
+---
+
+## Overview
+
+In this guide, we’ll be building a Discord app that lets server members play a *slightly*-enhanced version of rock paper scissors (with 7 choices instead of the usual 3).
+
+### Resources used in this guide
+- **[Github repository](https://github.com/shaydewael/discord-getting-started)** where the code from this guide lives along with additional feature-specific examples.
```

Comment:
As part of landing this PR, can we get this repository moved to the official Discord GitHub Organization?  I'm not entirely sure how that works 😆